applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -1,4 +1,16 @@
 import KratosMultiphysics
+import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
+import math
+
+def DotProduct(A,B):
```

Comment:
```suggestion
def _DotProduct(A,B):
```
This hides the function from becoming visible when you do `from ... import *`. It also communicates to the reader that this is not intended to be called externally.